tools/src/main/java/org/apache/kafka/tools/VerifiableConsumer.java

```diff
@@ -503,10 +503,17 @@ private static ArgumentParser argParser() {
                 .newArgumentParser("verifiable-consumer")
                 .defaultHelp(true)
                 .description("This tool consumes messages from a specific topic and emits consumer events (e.g. group rebalances, received messages, and offsets committed) as JSON objects to STDOUT.");
+        parser.addArgument("--bootstrap-server")
```

Comment:
We also want to deprecate the old options. Typically we just prefix the description with `DEPRECATED`. See for example, the `--zookeeper` argument use in `TopicCommand`. We could also print a warning, but it may be overkill.

Reply:
Updated